lib/gollum/views/helpers.rb

```diff
@@ -2,31 +2,63 @@
 
 module Precious
   module Views
+    module RouteHelpers
+      ROUTES = {
+        'gollum' => {
+          :last_commit_info => 'last-commit-info',
+          :latest_changes => 'latest_changes',
```

Comment:
I notice that `last-commit-info` is hyphenated, while `latest_changes` uses underscores. Any particular reason for this?

Reply:
I think this should actually be resolved in a separate PR.